gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/bin/gravitee

```diff
@@ -88,6 +88,11 @@ JAVA_OPTS="$JAVA_OPTS -XX:+HeapDumpOnOutOfMemoryError"
 # space for a full heap dump.
 #JAVA_OPTS="$JAVA_OPTS -XX:HeapDumpPath=$GRAVITEE_HOME/logs/heapdump.hprof"
 
+# JVM exits on the first occurrence of an out-of-memory error.
+if [ "$GIO_EXIT_ON_OOM_ERROR" = "true" ]; then
```

Comment:
I suggest flipping this logic, exiting on OOM unless this is explicitly set to false. Having the process hang around in an undefined state isn't useful. Defaults should be what's useful in most cases.